src/commonMain/kotlin/com/saveourtool/osv4k/OsvSchema.kt

```diff
@@ -770,3 +963,154 @@ enum class ReferenceType {
+data class PatchDetail(
+    @SerialName("patch_url")
+    @get:JsonProperty(
+        value = "patch_url",
+        namespace = "",
+        required = false,
+        index = -1,
+        defaultValue = "",
+        access = JsonPropertyAccess.AUTO
+    )
+    @JsonProperty(
+        value = "patch_url",
+        namespace = "",
+        required = false,
+        index = -1,
+        defaultValue = "",
+        access = JsonPropertyAccess.AUTO
+    )
+    val patchUrl: String? = null,
+    @SerialName("issue_url")
+    @get:JsonProperty(
+        value = "issue_url",
+        namespace = "",
+        required = false,
+        index = -1,
+        defaultValue = "",
+        access = JsonPropertyAccess.AUTO
+    )
+    @JsonProperty(
+        value = "issue_url",
+        namespace = "",
+        required = false,
+        index = -1,
+        defaultValue = "",
+        access = JsonPropertyAccess.AUTO
+    )
+    val issueUrl: String? = null,
+    @SerialName("main_language")
+    @get:JsonProperty(
+        value = "main_language",
+        namespace = "",
+        required = false,
+        index = -1,
+        defaultValue = "",
+        access = JsonPropertyAccess.AUTO
+    )
+    @JsonProperty(
+        value = "main_language",
+        namespace = "",
+        required = false,
+        index = -1,
+        defaultValue = "",
+        access = JsonPropertyAccess.AUTO
+    )
+    val mainLanguage: String? = null,
+    @JsonProperty(
+        value = "author",
+        namespace = "",
+        required = false,
+        index = -1,
+        defaultValue = "",
+        access = JsonPropertyAccess.AUTO
+    )
+    val author: String? = null,
+    @JsonProperty(
+        value = "committer",
+        namespace = "",
+        required = false,
+        index = -1,
+        defaultValue = "",
+        access = JsonPropertyAccess.AUTO
+    )
+    val committer: String? = null,
+    @JsonProperty(
+        value = "branches",
+        namespace = "",
+        required = false,
+        index = -1,
+        defaultValue = "",
+        access = JsonPropertyAccess.AUTO
+    )
+    val branches: List<String>? = null,
+    @JsonProperty(
+        value = "tags",
+        namespace = "",
+        required = false,
+        index = -1,
+        defaultValue = "",
+        access = JsonPropertyAccess.AUTO
+    )
+    val tags: List<String>? = null,
+)
+
+/**
+ * @property org the organization info of each contributor
+ * @property name the name of each contributor
+ * @property email the email address of each contributor
+ * @property contributions the description of the contributions each contributor has made
+ */
+@Serializable
+data class Contributor(
+    val org: String? = null,
+    val name: String? = null,
+    val email: String? = null,
+    val contributions: String? = null,
+)
+
+/**
+ * the confirmation type of this vulnerability record
+ */
+enum class ConfirmType {
+    manual_confirmed,
+    algorithm_confirmed,
+    double_confirmed,
```

Comment:
I'm to lazy, so decided to use not valid enum names, probably better to add `SerialName`, but will fix later